### PR TITLE
Swagger URL Related changes 

### PR DESCRIPTION
**Pull request by prashelke**
Thank you for your contribution.
Before submitting this PR, please make sure:
- [x] PR description and commit message should describe the changes done in this PR
- [x] Verify the PR is pointing to correct branch i.e. Release or Beta branch if the code fix is for specific release , else point it to master
- [x] Latest Code from master or specific release branch is merged to your branch
- [x] No unwanted\commented\junk code is included
- [x] No new warning upon build solution
- [ ] Code Summary\Comments are added to my code which explains what my code is doing
- [ ] Existing unit test cases are passed
- [ ] New Unit tests are added for your development
- [ ] Sanity Tests are successfully executed for New and Existing Functionality
- [ ] Verify that changes are compatible with all relevant browsers and platforms.
- [ ] After creating pull request there should not be any conflicts
- [ ] Resolve codacy comments
- [ ] Builds and checks are passed before PR is sent for review
- [ ] Resolve code review comments
- [ ] Help Libray document is updated accordingly for Feature changes.

### CODE REVIEW

File: Ginger/Ginger/ApplicationModelsLib/ModelParams Pages/ModelParamsPage.xaml.cs [hunk 1]
```diff
@@ -393,7 +393,7 @@ private void SyncParamsPendingDeleteWithBodyNodes(List<AppModelParameter> paramL
             bodyNodeSyncPage.ShowAsWindow();
         }
 
-        private void AddGlobalParametertoAPIGlobalParameterList(ObservableList<GlobalAppModelParameter> APIGlobalParamList, GlobalAppModelParameter GAMP)
+        public static void AddGlobalParametertoAPIGlobalParameterList(ObservableList<GlobalAppModelParameter> APIGlobalParamList, GlobalAppModelParameter GAMP)
```
Comment (Maheshkale447) [inline Ginger/Ginger/ApplicationModelsLib/ModelParams Pages/ModelParamsPage.xaml.cs:396]: Move this method to ModelParamsUtils

File: Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/AddAPIModelWizard.cs [hunk 2]
```diff
@@ -115,13 +124,50 @@ public override void Finish()
...
+                newModelGlobalParam.PlaceHolder = "{" + (!string.IsNullOrEmpty(newModelGlobalParam.PlaceHolder) ? newModelGlobalParam.PlaceHolder.Replace("{", "").Replace("}", "") : newModelGlobalParam.PlaceHolder) + "_Copy}";
+            }
+            ModelParamUtils.SetUniquePlaceHolderName(newModelGlobalParam);
+            newModelGlobalParam.OptionalValuesList.Add(new OptionalValue() { Value = customurl, IsDefault = true });
+            WorkSpace.Instance.SolutionRepository.AddRepositoryItem(newModelGlobalParam);
+            return newModelGlobalParam;
+        }
+
+
+        private void AddGlobalInModel(string zy)
```
Comment (Maheshkale447) [inline Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/AddAPIModelWizard.cs:143]: Please use correct name instead of zy

**Review by Maheshkale447 — CHANGES_REQUESTED** [review]

File: Ginger/Ginger/ApplicationModelsLib/ModelParams Pages/ModelParamsPage.xaml.cs (hunk 1 above)
Comment (prashelke) [reply Ginger/Ginger/ApplicationModelsLib/ModelParams Pages/ModelParamsPage.xaml.cs:396]: Done

File: Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/AddAPIModelWizard.cs (hunk 2 above)
Comment (prashelke) [reply Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/AddAPIModelWizard.cs]: Done

**Review by Maheshkale447 — APPROVED** [review]

**Pull request merged**